content/docs/enterprise/install/helm.mdx

```diff
@@ -123,6 +123,12 @@ This setup assumes an existing certificate solution using cert-manager, as descr
    image:
      pullUsername: pomerium/enterprise
      pullPassword: your-access-key
+   serviceMonitor:
+     enabled: true
+   metrics:
+     enabled: true
```

Comment:
unrelated changes?

Reply:
enables metrics collection from the console - added in https://github.com/pomerium/pomerium-helm/pull/336